client/packages.go

```diff
@@ -129,6 +130,16 @@ func (client *Client) List(names []string) ([]*Snap, error) {
 	return result, nil
 }
 
+// GetSections returns the list of existing snap sections in the store
+func (client *Client) GetSections() ([]string, error) {
```

Comment:
(also maybe interessting in this context in "effective go" which is a interesting read: https://golang.org/doc/effective_go.html#Getters)